examples/DotNetCore/MultiTenantApplicationSetup/README.md

```diff
@@ -0,0 +1,96 @@
+# Multi-tenant application setup
+
+The .NET Azure App Configuration Provider delivers configuration settings to the existing IConfiguration system. Multi-tenant applications using the provider can take advantage of IConfiguration features to retrieve tenant information from requests. The following instructions will demonstrate one way to achieve this application setup.
+
+1. Define the per tenant settings in App Configuration as key-values using key prefixes.
+
+    #### Tenant 1
+
+    | Key | Value |
+    |---- |-------|
+    | Contoso:Name | Contoso Corp. |
+    | Contoso:Color | Azure |
+
+    #### Tenant 2
+
+    | Key | Value |
```

Comment:
Another name we can use similar to Contoso is Fabrikam. I'd recommend that instead of `ExampleTenant` which isn't analogous to contoso.